roxctl/deployment/check/check.go

```diff
@@ -172,6 +175,12 @@ func (d *deploymentCheckCommand) Construct(_ []string, cmd *cobra.Command, f *pr
 }
 
 func (d *deploymentCheckCommand) Validate() error {
+	if d.cluster != "" && d.namespace == defaultNamespace {
+		d.env.Logger().WarnfLn("No custom namespace provided. Any deployments without an explicit namespace will be defaulted to \"default\". Overwrite the default with --namespace/-n [NAMESPACE]\n")
```

Comment:
```suggestion
		d.env.Logger().WarnLn("Deployments without a namespace in the spec will be enhanced with context information for the \"default\" namespace. This can be changed with setting --namespace/-n")
```

Options for the value:
- `<name>`
- `NAME`

Reply:
Done